### PR TITLE
Feature/organization ordering

### PR DESCRIPTION
**Pull request by Bluesmile82**
This PR orders the `Organization type` and `Organization theme` by a new `order` attribute. This attribute is required as it won't be clear how we will handle undefined values and the default is 0. This information is added on the documentation on the CMS display. The values with higher order will appear below in the list and it will also be ordered alphabetically.

https://vizzuality.atlassian.net/browse/ORC-400?atlOrigin=eyJpIjoiYTIyMmQ0YjRmOTkxNGRmNTgyYzk0NzAxZGVmMzYyY2IiLCJwIjoiaiJ9

### CODE REVIEW

File: cms/types/generated/contentTypes.d.ts [hunk 1]
```diff
@@ -1199,6 +1199,7 @@ export interface ApiOrganizationThemeOrganizationTheme
   };
   attributes: {
     name: Attribute.String & Attribute.Required & Attribute.Unique;
+    order: Attribute.Integer & Attribute.DefaultTo<1>;
```
Comment (clementprdhomme) [inline cms/types/generated/contentTypes.d.ts:1202]: According to the rest of the code (especially `cms/src/api/organization-theme/content-types/organization-theme/schema.json`), the default should be 0, no?

**Review by clementprdhomme — COMMENTED** [review]
There seem to be a lot of changes (especially names) in the CMS. Were those changes intentional?

**Comment by Bluesmile82** [comment]
I updated the config but I'm not sure why we don't have default value on the config anymore. It is on my cms

File: cms/types/generated/contentTypes.d.ts (hunk 1 above)
Comment (Bluesmile82) [reply cms/types/generated/contentTypes.d.ts]: I finally changed it manually as I couldn't get it generated from the CMS interaction or exporting the config. I'm not sure if this actually affects the CMS behavior, though.

**Review by clementprdhomme — APPROVED** [review]

**Pull request merged**